File: herencia-dto/SergioDeIscar-herencia-java/SergioDeIscar-herencia-java/src/main/java/org/example/dto/PersonasDto.java
package org.example.dto;

import org.simpleframework.xml.ElementList;
import org.simpleframework.xml.Root;

import java.io.Serializable;
import java.util.List;

@Root(name = "personas")
public class PersonasDto implements Serializable {
    public PersonasDto(List<PersonaDto> personas) {
        this.personas = personas;
    }
    @ElementList(name = "persona", inline = true)
    private List<PersonaDto> personas;

    public List<PersonaDto> getPersonas() {
        return personas;
    }
}

File: herencia-dto/DanielPaz-HerenciaDTO/HerenciaDTO-Kotlin/HerenciaDTO_Kt/src/main/kotlin/dto/PersonasDto.kt
package dto


import org.simpleframework.xml.ElementList
import org.simpleframework.xml.Root


@Root(name = "personas")
data class PersonasDto(

    @field:ElementList(name = "persona", inline=true)
    @param:ElementList(name = "persona", inline=true)
    val personas:  List<PersonaDto>
)